src/python/pants/backend/experimental/javascript/register.py

```diff
@@ -17,11 +21,19 @@ def rules() -> Iterable[Rule | UnionRule]:
     return (
         *nodejs.rules(),
         *tailor.rules(),
+        *lockfile.rules(),
+        *package_rules(),
+        *run_rules(),
```

Comment:
I had to make a graph of rules to satisfy myself that this includes all of the util rules.
- `package_rules` and `run_rules` include `install_node_package.rules`
- `lockfile.rules` includes `nodejs_project_environment.rules`

So, starting from those, these include all the other rules.
- `install_node_package.rules()`
    - `nodejs.rules()`
    - `nodejs_project_environment.rules()`
        - `nodejs.rules()`
        - `resolve.rules()`
            - `nodejs_project.rules()`
                - `package_json.rules()`
        - `package_json.rules()`
    - `dependency_inference.rules()`
        - `package_json.rules()`
        - `import_parser.rules()`
            - `nodejs.rules()`

So, I guess nothing else needs to be done in this file. I'll just leave this here for anyone that is doing some spelunking 😄.